src/commands/etherfi.ts

```diff
@@ -102,17 +102,34 @@ etherfi
     let expectedNonce = nonce;
 
     for (let keysharesChunk of [...chunks(keyshares, chunkSize)]) {
-      // update nonce
-      console.info(`Obtaining owner nonce`);
-      nonce = await getOwnerNonceFromSubgraph(process.env.SAFE_ADDRESS);
-      console.info(`Owner nonce: ${nonce}`);
-      // expected to be the same on first loop, but important on following ones
-      if (nonce !== expectedNonce) {
-        console.error(
-          "Nonce has not been updated since last successful transaction!"
+
+      try {
+        await retryWithExponentialBackoff(verifyUpdatedNonce, {nonce, expectedNonce, ownerAddress: process.env.SAFE_ADDRESS}, {
+          retries: 3,
+          factor: 2,
```

Comment:
unsure what factor is here?

apart from that LGTM

Reply:
This is the exponential factor for the backoff.